definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField-nonprod.json

```diff
@@ -19,5 +19,33 @@
     "CaseFieldID": "d11",
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "latestDraftDirectionOrder",
+    "UserRole": "caseworker-divorce-financialremedy-judiciary",
```

Comment:
don't use this file. I want to get rid of it but I don't know what epic the d11 field falls under. if you want to toggle this off then put it in a nonprod file for the epic. your story suggests that we want it to be CI/CD though so should this be in the base file?